enterprise/backend/src/metabase_enterprise/audit/pages/queries.clj

```diff
@@ -54,8 +54,13 @@
 (s/defn ^:internal-query-fn table
   "A list of all questions."
   ([]
-   (table nil))
-  ([query-string :- (s/maybe s/Str)]
+   (table nil nil "card.name" "asc"))
+  ([questionFilter :- (s/maybe s/Str)]
+   (table questionFilter nil "card.name" "asc"))
```

Comment:
This is all internal data so much more common to use `:card.name` and `:asc` and `(s/enum :asc :desc)` than string keys. A docstring that explains some of this would be quite helpful to let people know how to call this and what arguments to give it without looking at the source.

Reply:
Trouble with this is that the input stuff is handled by magic in the middleware and it would complicate the middleware and sorta explode the scope to have the middleware turn certain stuff into keywords

Reply:
Docstring coming tho, I guess